packages/swapper/src/api.ts

```diff
@@ -83,9 +83,12 @@ export interface TradeQuote<C extends SupportedChainIds> extends TradeBase<C> {
 }
 
 export interface Trade<C extends SupportedChainIds> extends TradeBase<C> {
+  receiveAddress: string
+}
+
+export interface ZrxTrade<C extends SupportedChainIds> extends Trade<C> {
```

Comment:
please comment in https://discord.com/channels/554694662431178782/886041582422356018/978367019651047454 to find a better home for `SupportedChainIds` before we spread this pattern further

Reply:
Based on @majorhayes response in the discord discussion I feel like we need to talk about it further and not necessarily make the decision with this PR